pkg/apis/componentconfig/types.go

```diff
@@ -596,6 +596,15 @@ type KubeSchedulerConfiguration struct {
 	FailureDomains string
 	// leaderElection defines the configuration of leader election client.
 	LeaderElection LeaderElectionConfiguration
+	// PolicyConfigMapName is the name of the ConfigMap object that specifies
+	// the scheduler's policy config. If UseLegacyPolicyConfig is true, scheduler
+	// uses PolicyConfigFile. If UseLegacyPolicyConfig is false and
+	// PolicyConfigMapName is not empty, the ConfigMap object with this name must
+	// exist before scheduler initialization.
+	PolicyConfigMapName string
```

Comment:
Also do we need to know it's a configmap?  Isn't it just loading a file and the config map is an implementation detail.